src/v/raft/configuration_manager.cc

```diff
@@ -211,9 +212,26 @@ const group_configuration& configuration_manager::get_latest() const {
     vassert(
       !_configurations.empty(),
       "Configuration manager should always have at least one configuration");
+    if (_configuration_force_override) [[unlikely]] {
```

Comment:
What is the mechanism that allows `controller_backend` to understand that the latest configuration is still ephemeral (and therefore the update can't be marked as finished yet)?

Reply:
the `controller_backend` is waiting for leader to be elected and first configuration batch is applied.